app/controllers/transfer_packages_controller.rb

```diff
@@ -77,12 +77,15 @@ def find_box
     @boxes = Box
       .within(@navigation_context.entity, @navigation_context.exclude_subsites)
       .autocomplete(uuid)
-      .order("created_at DESC")
+      .order("boxes.created_at DESC")
       .count_samples
       .limit(5)
 
     @boxes = check_access(@boxes, READ_BOX)
 
+    # blocks resending the same box
+    @boxes = @boxes.reject { |box| BoxTransfer.where(box: box).present? }
```

Comment:
I tried it, but it returns an error. I will put here what happened, but was some conflict with other filters, because of the samples count, this query is grouped, or something like this.

Reply:
The only way I managed to do it was this, as it is a query has a `limit 5` and almost always the user will paste a hash Id, I imagine that is not a big problem to make an additional query on this specific case.

Reply:
I apologize, my suggestions was full of typos 🙈 
This is correct:
```suggestion
    @boxes = @boxes.left_joins(:box_transfers).where(box_transfers: {id: nil})
```

Reply:
Yeah, I don't know what I was doing wrong when I did this task, but I didn't get any errors this time. It works. In fact I was studying the ActiveRecord functionality, so probably I did something wrong.

```
@boxes = Box
      .within(@navigation_context.entity, @navigation_context.exclude_subsites)
      .left_joins(:box_transfers)
      .where(box_transfers: {id: nil})
      .autocomplete(uuid)
      .order("boxes.created_at DESC")
      .count_samples
      .limit(5)
```